src/assets/css/layout/header.css

```diff
@@ -1,8 +1,7 @@
 .hero {
-    background: url("/src/assets/images/Image.png") no-repeat;
+    background: url("/assets/images/Image.png") no-repeat;
```

Comment:
what is Image.png naming not make sense

Reply:
I just found out when I working on inspired section, hehe, I will modify it, thanks <3